scripts/lib/canary.sh

```diff
@@ -4,8 +4,13 @@
 
 SECONDS=0
 
+# Fallback to KUBE_CONFIG_PATH if KUBECONFIG is not set
+
+source ./set_kubeconfig.sh
```

Comment:
Using `./set_kubeconfig.sh`make dependent on the path.
The previous version better with the change suggested; so that we could remove that whole condition later.

If you still prefer this version, please change it to use `SCRIPT_DIR` reference (like SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )") or you use the previous rev with just the change suggested too.

Sorry for this back and forth, my idea was to temporarily remove the dependency on KUBE_CONFIG_PATH path, while not breaking any runners and then getting rid of it. Adding another script doesn't help much.

Reply:
@orsenthil can you check updated commit